app/javascript/components/layout/CustomTable.js

```diff
@@ -205,8 +205,8 @@ class CustomTable extends React.Component {
             </tbody>
           </Table>
         </div>
-        <div className="d-flex justify-content-between">
-          <Form inline className="align-middle">
+        <Row>
+          <Col xs="24" className="col-xxl-24 col-xxxl-12">
```

Comment:
You actually don't need the xs breakpoint here because it is the same as the xxl one